challenge-mock-data.json

```diff
@@ -0,0 +1,421 @@
+          "points": 102,
+          "approved": 22,
+          "accuracy": 10.11
+        },
+        {
+          "position": 4,
+          "name": "catherine4",
+          "points": 103,
+          "approved": 23,
+          "accuracy": 10.12
+        },
+        {
+          "position": 5,
+          "name": "catherine5",
+          "points": 104,
+          "approved": 24,
+          "accuracy": 10.13
+        },
+        {
+          "position": 6,
+          "name": "catherine6",
+          "points": 105,
+          "approved": 25,
+          "accuracy": 10.14
+        },
+        {
+          "position": 7,
+          "name": "catherine7",
+          "points": 106,
+          "approved": 26,
+          "accuracy": 10.15
+        },
+        {
+          "position": 8,
+          "name": "catherine8",
+          "points": 107,
+          "approved": 27,
+          "accuracy": 10.16
+        },
+        {
+          "position": 9,
+          "name": "catherine9",
+          "points": 108,
+          "approved": 28,
+          "accuracy": 10.17
+        },
+        {
+          "position": 10,
+          "name": "catherine10",
+          "points": 109,
+          "approved": 29,
+          "accuracy": 10.18
+        }
+      ]
+    },
+    "validated": {
+      "member": [
+        {
+          "position": 1,
+          "name": "can1",
+          "points": 110,
+          "approved": 30,
+          "accuracy": 9.09
+        },
+        {
+          "position": 2,
+          "name": "can2",
+          "points": 111,
+          "approved": 31,
+          "accuracy": 9.1
+        },
+        {
+          "position": 3,
+          "name": "can3",
+          "points": 112,
+          "approved": 32,
+          "accuracy": 9.11
+        },
+        {
+          "position": 4,
+          "name": "can4",
+          "points": 113,
+          "approved": 33,
+          "accuracy": 9.12
+        },
+        {
+          "position": 5,
+          "name": "can5",
+          "points": 114,
+          "approved": 34,
+          "accuracy": 9.13
+        },
+        {
+          "position": 6,
+          "name": "can6",
+          "points": 115,
+          "approved": 35,
+          "accuracy": 9.14
+        },
+        {
+          "position": 7,
+          "name": "can7",
+          "points": 116,
+          "approved": 36,
+          "accuracy": 9.15
+        },
+        {
+          "position": 8,
+          "name": "can8",
+          "points": 117,
+          "approved": 37,
+          "accuracy": 9.16
+        },
+        {
+          "position": 9,
+          "name": "can9",
+          "points": 118,
+          "approved": 38,
+          "accuracy": 9.17
+        },
+        {
+          "position": 10,
+          "name": "can10",
+          "points": 119,
+          "approved": 39,
+          "accuracy": 9.18
+        }
+      ]
+    }
+  },
+  "topTeams": [
+    {
+      "position": 1,
+      "name": "SAP1",
+      "logo": "base641...",
```

Comment:
Not a blocker obviously, but I want to make sure we're on the same page. I was thinking these would eventually be external URLs from the `logo_url` column of our database (or alternatively, some constants file on the front-end), not base64 encoded. Is that right?

Reply:
Yes, that is also my thought.